### PR TITLE
pom and base classes

### PR DESCRIPTION
**Pull request by sharksystems**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,38 @@
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
+
+### IntelliJ IDEA ###
+.idea/modules.xml
```
Comment (samsometro) [inline .gitignore:8]: .idea

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>TSBank</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>20</maven.compiler.source>
```
Comment (samsometro) [inline pom.xml:12]: Correct properties and dependencies

File: src/test/java/Tests/TestInit.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+public class TestInit {
+
+    WebDriver driver;
+    @BeforeMethod
+    public void setup (){
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+    }
+    @AfterMethod
+    public void bquit () {
```
Comment (samsometro) [inline src/test/java/Tests/TestInit.java]: tearDownDriver()

File: src/test/java/Tests/TestInit.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        driver.quit();
+    }
+    public void goToSite (){
```
Comment (samsometro) [inline src/test/java/Tests/TestInit.java:21]:
```suggestion
    public void env (){
```

**Comment by sharksystems** [comment]
fixed in 2nd commit

**Review by samsometro — APPROVED** [review]

**Pull request merged**